File: iniciante/javascript/1101.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function soma(a, b){
    let menor
    let maior
    let soma = 0
    let saida = []
    let count
    if(a < b){
        menor = a
        maior = b
    }else{
        menor = b
        maior = a
    }
    
    for(let i = menor, count = 0; i <= maior; i++, count++){
        soma += i
        saida[count] = i
        if(i == 0)
            return 0
    }
    console.log(`${saida.join(' ')} Sum=${soma}`)
    return 1
}


do{
    entradas = lines.shift().split(' ')
    a = parseInt(entradas[0])
    b = parseInt(entradas[1].replace('\r',''))
}while(soma(a, b) != 0)

File: iniciante/python/1101.py
# -*- coding: utf-8 -*-

while True:
    soma = 0
    entrada = input().split(' ')
    if int(entrada[0]) > 0 and int(entrada[1]) > 0:
        entrada = [int(x) for x in entrada]
        entrada.sort()
        for x in range(entrada[0], entrada[1] + 1):
            soma += x
            print(x, end=' ')
    else:
        break
    print(f'Sum={soma}')
